.github/labeler.yml

```diff
@@ -1,26 +1,34 @@
 # Add 'kernel' label to any change within Connectors, Extensions, Skills, and tests directories
 kernel:
-  - "dotnet/src/Connectors/**/*"
-  - "dotnet/src/Extensions/**/*"
-  - "dotnet/src/Skills/**/*"
-  - "dotnet/src/IntegrationTests/**/*"
-  - "dotnet/src/SemanticKernel.UnitTests/**/*"
+  - dotnet/src/**/*
+  - python/**/*
+  - java/**/*
 
 # Add 'kernel.core' label to any change within the 'SemanticKernel', 'SemanticKernel.Abstractions', or 'SemanticKernel.MetaPackage' directories
 kernel.core:
   - dotnet/src/SemanticKernel/**/*
   - dotnet/src/SemanticKernel.Abstractions/**/*
   - dotnet/src/SemanticKernel.MetaPackage/**/*
-
-# Add 'python' label to any change within the 'python' directory
-python:
-  - python/**/*
+  - python/semantic_kernel**/*
```

Comment:
Won't this include connectors/skill only changes in python?

Reply:
yes, I see that as part of the core. I think it would help renaming the labels, there is a lot of overlap between the definition of "kernel" and "kernel core"